src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -447,7 +447,12 @@ pub const NI_NUMERICSCOPE: ::c_int = 0x00000020;
 pub const Q_GETQUOTA: ::c_int = 0x700;
 pub const Q_SETQUOTA: ::c_int = 0x800;
 
+macro_rules! MAP_ALIGNED {
+    ($n:tt) => {($n as ::c_int) << 24}
+}
+
 pub const MAP_GUARD: ::c_int = 0x00002000;
+pub const MAP_ALIGNED_SUPER: ::c_int = MAP_ALIGNED!(1);
```

Comment:
I'd avoid using such a macro to simplify code as possible. The use of `MAP_ALIGNED` seems quite a few (`MAP_ALIGNMENT_MASK` and `MAP_ALIGNED_SUPER`) so I think it's okay just to hardcode `1 << 24`.